team_wizz.js

```diff
@@ -1 +1,33 @@
 // Fetch all routes for the airline and insert into #wizz
+
+(async () => {
```

Comment:
Can you convert this into a function?
I'd like to see that it is separated, see next comment on details.